Dockerfile

```diff
@@ -1,24 +1,39 @@
-FROM node:18-alpine
+FROM --platform=$BUILDPLATFORM node:18-alpine AS build
+ARG BUILDPLATFORM
```

Comment:
This is a built-in Docker arg that resolves to the native platform of the builder: https://docs.docker.com/build/building/multi-platform/#cross-compilation

Reply:
I didn't know that, good to know